src/lib/radio/radio.spec.ts

```diff
@@ -218,6 +218,17 @@ describe('MdRadio', () => {
       expect(radioInstances.every(radio => !radio.checked)).toBe(true);
     });
 
+    it('should not have a ripple on disabled radio buttons', () => {
+      let rippleElement = radioNativeElements[0].querySelector('[md-ripple');
```

Comment:
It looks like you're missing the end square bracket in the selector? Same on line 228.